### PR TITLE
Expose learned macs via Forch varz

### PR DESCRIPTION
**Pull request by didovesei**
Expose learned macs with swith, port, ip, VLAN and ACLs via Forch varz.

Also shared a doc with output examples.

### CODE REVIEW

File: forch/faucet_state_collector.py [hunk 1]
```diff
@@ -635,19 +636,19 @@ def _fill_acls_behavior(self, switch_name, acls_map_list, acls_config,
             rules_map_list = acl_map.setdefault('rules', [])
 
             for rule_config in acl_config.rules:
-                cookie = str(rule_config.get('cookie'))
+                num_cookie = rule_config.get('cookie')
```
Comment (grafnu) [inline forch/faucet_state_collector.py:639]: cookie_num --- num_cookie would be a count of the number of cookies.

File: forch/forch_metrics.py [hunk 2]
```diff
@@ -80,6 +80,10 @@ def _add_vars(self):
                       'No. of RADIUS query responses received from server', Counter)
         self._add_var('process_state', 'Current process state', Gauge, labels=['process'])
 
+        learned_mac_help_text = 'MAC address stored as 64bit number to switch anme, port, VLAN, '\
```
Comment (grafnu) [inline forch/forch_metrics.py:83]: don't use line continuation, this string seems long to have here -- where does it show up?

File: forch/faucet_state_collector.py (hunk 1 above)
Comment (didovesei) [reply forch/faucet_state_collector.py]: Done

File: forch/forch_metrics.py (hunk 2 above)
Comment (didovesei) [reply forch/forch_metrics.py]: This is the HELP message that will be shown when scraping the metrics. Updated the message to better describe the new format.

**Pull request merged**